Karatsuba_Algo/explanation.md

```diff
@@ -0,0 +1,20 @@
+# Karatsuba Multiplication
```

Comment:
Change filename to Readme.md. It is standard name used mostly everywhere.

Reply:
Not done.

Reply:
I have changed the filename. @asha15 and @jainaman224, Please review.